### PR TITLE
task #87 - ci: Added CI workflow

### PR DESCRIPTION
**Pull request by carvilgar1**
workflow must be in the main branch to be triggered

### CODE REVIEW

File: .github/workflows/eventus-cicd.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup node 16
+      uses: actions/setup-node@v2
+      with:
+        node-version: '16'
+    - name: Run frontend test
+      run: 
+        npm install
+        npm test
```
Comment (laur2000) [inline .github/workflows/eventus-cicd.yml:59]: Los tests se ejecutan con el comando `npm run test`

**Review by laur2000 — CHANGES_REQUESTED** [review]

**Review by JSnow11 — CHANGES_REQUESTED** [review]
## Review
- added some comments about workflow triggers
- we can also add the codacy report, I'm creating a secret named `CODACY_KEY`

**Comment by carvilgar1** [comment]
This branch should also be merged with develop

**Comment by carvilgar1** [comment]
image
Now working with java

**Comment by carvilgar1** [comment]
image
image

Now the whole project is covered

**Review by JSnow11 — APPROVED** [review]
## Review
ready to be merged

**Review by laur2000 — APPROVED** [review]

**Pull request merged**